purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -930,6 +971,45 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         }
     }
 
+    @JvmSynthetic internal fun updatePendingPurchaseQueue() {
+        if (billingWrapper.isConnected()) {
+            debugLog("[QueryPurchases] Updating pending purchase queue")
+            executorService.execute {
+                val querySUBS = billingWrapper.queryPurchases(BillingClient.SkuType.SUBS)
+                val queryINAPP = billingWrapper.queryPurchases(BillingClient.SkuType.INAPP)
```

Comment:
`queryINAPP` and `querySUBS` reads more like a verb phrase than a noun phrase. It's not clear from the variable name what they are.